src/styles/header/variables.scss

```diff
@@ -0,0 +1,4 @@
+$HTMLfont-family: roboto, sans-serif;
```

Comment:
Why we have two files of variables?

Reply:
one of them includes card-block's variables and another  for header-block

Reply:
Clean up all your variables. Don't redeclare the same variables multiple times.